dev/TreeView/TestUI/TreeViewPage.xaml.cs

```diff
@@ -162,7 +162,13 @@ private void TestTreeView_ItemInvoked(TreeView sender, TreeViewItemInvokedEventA
             }
         }
 
-        private void TestTreeViewContentMode_ItemInvoked(TreeView sender, TreeViewItemInvokedEventArgs args)
+        private void TestTreeView_SelectionChanged(TreeView sender, SelectionChangedEventArgs args)
```

Comment:
Please add an interaction test to exercise this new API.

Reply:
Done.